webrtc.html

```diff
@@ -1683,18 +1683,18 @@
             set and the set of certificates differs from the ones used
```

Comment:
Don't we also want to remove "is set" from here? And add a default value (`[]`)? I may be getting my `setConfiguration` issues confused.